src/components/PoolOverview.tsx

```diff
@@ -91,6 +99,24 @@ function PoolOverview({
             <span className="label">{t("currencyReserves")}</span>
             <span>{`$${formattedData.reserve}`}</span>
           </div>
+          {formattedData.utilization && (
+            <div className="volume">
+              <span className="label">Utilization</span>
+              <span>{formattedData.utilization}</span>
+            </div>
+          )}
+          {formattedData.apy && (
```

Comment:
Could you change the order of these so they're in the order APY, APR, Reserves, 24h Volume?